cdap-client/src/main/java/co/cask/cdap/client/config/ClientConfig.java

```diff
@@ -115,14 +116,29 @@ public URL resolveURLNoVersion(String path) throws MalformedURLException {
   }
 
   /**
+   * @deprecated use {@link #resolveNamespacedURLV3(NamespaceId namespace, String path)} instead
+   *
    * Resolves a path against the target CDAP server with the provided namespace, using V3 APIs
    *
    * @param path Path to the HTTP endpoint. For example, "apps" would result
    *             in a URL like "http://example.com:11015/v3/&lt;namespace&gt;/apps".
    * @return URL of the resolved path
    * @throws MalformedURLException
    */
+  @Deprecated
```

Comment:
Yes, because this method only returns the result from calling `resolveNamespacedURLV3` and that method only uses the String value of namespace to form a new URL